frontend/src/lib/lemon-ui/LemonSelectMultiple/LemonSelectMultiple.stories.tsx

```diff
@@ -5,34 +5,31 @@ import { useState } from 'react'
 import { ProfilePicture } from '../ProfilePicture'
 import { LemonSelectMultiple, LemonSelectMultipleProps } from './LemonSelectMultiple'
 
+const names = ['ben', 'marius', 'paul', 'tiina', 'tim', 'james', 'neil', 'tom', 'paul', 'thomas']
```

Comment:
Do we want to dedupe options in any way? Not super important but just noticed it because we have Paul here twice

Reply:
Nah this is only relevant in a test case like this. I'll just fix the list